packages/chopsticks/src/plugins/index.ts

```diff
@@ -1,45 +1,51 @@
 import { Handlers } from '@acala-network/chopsticks-core'
-import { lstatSync, readdirSync } from 'fs'
+import { existsSync, lstatSync, readdirSync } from 'fs'
 import _ from 'lodash'
 import type { Argv } from 'yargs'
 
 import { defaultLogger } from '../logger.js'
 
 const logger = defaultLogger.child({ name: 'plugin' })
 
-export const pluginHandlers: Handlers = {}
+export const rpcPluginHandlers: Handlers = {}
 
 // list of plugins directory
 const plugins = readdirSync(new URL('.', import.meta.url)).filter((file) =>
   lstatSync(new URL(file, import.meta.url)).isDirectory(),
 )
 
+// find all rpc exports
+export const rpcPluginMethods = readdirSync(new URL('.', import.meta.url))
+  .filter((file) => existsSync(new URL(`${file}/rpc.ts`, import.meta.url)))
```

Comment:
this won't work when it's build

Reply:
don't use exists. use lstatSync instead and see if there's a filename with rpc

Reply:
both `exist` and `lstatSync` won't work, they both require full filename with extension, so I changed to filter by file name instead